src/main/java/com/google/cloud/genomics/dataflow/utils/ShardedBAMWritingOptions.java

```diff
@@ -0,0 +1,18 @@
+package com.google.cloud.genomics.dataflow.utils;
```

Comment:
@deflaux 
So one argument for separate options is when somebody wants to reuse a bunch of transforms that use particular options. In the case of a separate file they only depend on the options, while in the case where options are bound to the pipeline code itself they have to depend on the file which also houses specific pipeline implementations.